Gemfile

```
@@ -8,7 +8,9 @@ gem 'rubyzip', '~> 1.2.2'
 gem 'mysql2', '~>0.3.20'
 gem 'omniauth'
 gem 'omniauth-shibboleth'
-gem 'recaptcha'
+# pin recaptcha to work with Ruby 2.1 and Rails 3
+# some work is required to upgrade to 4.9. So pinned it to 3.x for now
+gem 'recaptcha', '~>3'
```

Then we know what version we pinned it to and dependabot can alert us to version bumps.
```suggestion
gem 'recaptcha', '3.4.0'
```